src/components/Form/From/AvailableActions/index.tsx

```diff
@@ -1,13 +1,14 @@
 interface AvailableActionsProps {
   max?: number;
   setValue?: (n: number) => void;
+  hideAvailable?: boolean;
```

Comment:
I was confused when first reading it, let's maybe be more specific
```suggestion
  hideAvailableBalance?: boolean;
```